<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carlos</class>
 <widget class="QDialog" name="carlos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(51, 51, 51)</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(54, 54, 54);
border-bottom: 1px solid rgb(245, 245, 245);
color: rgb(245, 245, 245);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_7">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>70</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_12">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_13">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_10">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_11">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_14">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_15">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_16">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_17">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>17</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_18">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>18</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_19">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>19</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_20">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_21">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>310</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: #6cc644;
        border: 1px solid #f5f5f5;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {background-color: #5eaa3b;
    }
    QPushButton:pressed {background-color: #589f37;
    }</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_resv">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: #f5f5f5;
        border: 1px solid #6e5494;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {
        background-color: #e9e9e9;
    }
    QPushButton:pressed {
        background-color: #d6d6d6;
    }</string>
   </property>
   <property name="text">
    <string>Reservaciones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_salir">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>440</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: #f5f5f5;
        border: 1px solid #6e5494;
        border-radius: 10px;
        padding: 10px 20px;
    }
    QPushButton:hover {
        background-color: #e9e9e9;
    }
    QPushButton:pressed {
        background-color: #d6d6d6;
    }</string>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f5f5f5;
border: 1px solid #6e5494;
border-radius: 10px;
padding: 10px 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #6cc644;
border: 1px solid #f5f5f5;
border-radius: 10px;
padding: 10px 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #bd2c00;
border: 1px solid #f5f5f5;
border-radius: 10px;
padding: 10px 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f6c40f;
border: 1px solid #f5f5f5;
border-radius: 10px;
padding: 10px 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f5f5f5;
border: 1px solid #f5f5f5;
border-radius: 10px;
padding: 10px 20px;</string>
   </property>
   <property name="text">
    <string>Disponible</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f5f5f5;
border: 1px solid #f5f5f5;
border-radius: 10px;
padding: 10px 20px;</string>
   </property>
   <property name="text">
    <string>Reservado</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>-15</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f5f5f5;
border: 1px solid #f5f5f5;
border-radius: 10px;
padding: 10px 20px;</string>
   </property>
   <property name="text">
    <string>Ocupado</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>-15</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="Title_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(54, 54, 54);
border-bottom: 1px solid rgb(245, 245, 245);
border-left: 1px solid rgb(245, 245, 245);
color: rgb(245, 245, 245);</string>
   </property>
   <property name="text">
    <string>Recepción</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="Title_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(54, 54, 54);
border-bottom: 1px solid rgb(245, 245, 245);
color: rgb(245, 245, 245);</string>
   </property>
   <property name="text">
    <string>Encargado:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="emp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(54, 54, 54);
border-bottom: 1px solid rgb(245, 245, 245);
color: rgb(245, 245, 245);</string>
   </property>
   <property name="text">
    <string>************</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
